x-pack/plugins/actions/server/builtin_action_types/pagerduty.ts

```diff
@@ -218,11 +217,23 @@ async function executor(
 
 const AcknowledgeOrResolve = new Set([EVENT_ACTION_ACKNOWLEDGE, EVENT_ACTION_RESOLVE]);
 
-function getBodyForEventAction(actionId: string, params: ActionParamsType): any {
+function getBodyForEventAction(actionId: string, params: ActionParamsType): unknown {
   const eventAction = params.eventAction || EVENT_ACTION_TRIGGER;
   const dedupKey = params.dedupKey || `action:${actionId}`;
 
-  const data: any = {
+  const data: {
+    event_action: ActionParamsType['eventAction'];
+    dedup_key: string;
+    payload?: {
+      summary: string;
+      source: string;
+      severity: string;
+      timestamp?: string;
+      component?: string;
+      group?: string;
+      class?: string;
+    };
+  } = {
```

Comment:
Not that this is a change strictly to typing, not actual object, so compile-time addition, not runtime.